### ISSUE TITLE
Support for new P-Code operation in Ghidra 10.3

### ISSUE THREAD

**Issue by Enkelmann** [fkie-cad/cwe_checker#411]
Short version: If you run into crashes due to the new P-Code operation `lzcount`, please switch to the older Ghidra version 10.2.3 for the time being.

Long version: Ghidra 10.3 was released a few days ago and contains some exciting new features that we want to use to improve the cwe_checker! It also contains a new P-Code operation named `lzcount`, which is currently not supported by the cwe_checker. Since we are currently in the midst of a larger refactoring of the P-code extractor code, we decided to postpone implementation of this new P-code operation until the refactoring is finished. So until then you might run into crashes when using Ghidra 10.3 together with the cwe_checker. I will keep this issue open until support for `lzcount` has been implemented.

**Comment by Enkelmann** [comment]
UPDATE: We implemented support for `lzcount` in PR #444 without waiting on the big refactoring branch. Mostly, because we still are not sure when the refactoring will land in the master branch. So now the cwe_checker should work with recent Ghidra versions again.
